scripts/src/java/org/oppia/android/scripts/label/AccessibilityLabelCheck.kt

```diff
@@ -0,0 +1,156 @@
+ * @return whether the label is present or not
+ */
+private fun checkIfActivityHasMissingLabel(
+  activityNode: Node,
+  accessibilityLabelExemptionList: List<String>
+): Boolean {
+  val attributesList = activityNode.getAttributes()
+  val activityPath = attributesList.getNamedItem("android:name").getNodeValue()
+  if (
+    activityPath
+      .removePrefix(".")
+      .replace(".", "/") in accessibilityLabelExemptionList
+  ) {
+    return false
+  }
+  return attributesList.getNamedItem("android:label") == null
+}
+
+/**
+ * The [nodeList] is not iterable. This helper
+ * function converts it to List<Node>.
+ *
+ * @param nodeList instance of NodeList
+ * @return a list of nodes
+ */
+private fun convertNodeListToListOfNode(nodeList: NodeList): List<Node> {
+  return IntStream.range(0, nodeList.getLength())
+    .mapToObj(nodeList::item)
+    .collect(Collectors.toList())
+}
+
+/**
+ * Logs the failures for accessibility label check.
+ *
+ * @param matchedNodes a list of nodes having missing label
+ * @param repoPath path of the repo to be analyzed
+ */
+private fun logFailures(
+  matchedNodes: List<Node>,
+  repoPath: String,
+  accessibilityLabelExemptiontextProto: String
+) {
+  val pathPrefix = "${repoPath}app/src/main/java/org/oppia/android"
+  if (matchedNodes.isNotEmpty()) {
+    println("Accessiblity labels missing for Activities:")
+    matchedNodes.sortedBy {
+      it.getAttributes()
+        .getNamedItem("android:name")
+        .getNodeValue()
+    }.forEach { node ->
+      println(
+        "- $pathPrefix" +
+          "${
+          node.getAttributes()
+            .getNamedItem("android:name")
+            .getNodeValue()
+            .replace(".", "/")
+          }"
+      )
+    }
+    println("If this is correct, please update $accessibilityLabelExemptiontextProto.textproto")
+    println(
+      "Note that, in general, all Activities should have labels. If you choose to add an" +
+        " exemption, please specifically call this out in your PR description."
+    )
+    println()
+  }
+}
+
+/**
+ * Loads the test file exemptions list to proto.
+ *
+ * @param accessibilityLabelExemptiontextProto the location of the accessibility label exemption
+ *     textproto file.
+ * @return proto class from the parsed textproto file
+ */
+private fun loadAccessibilityLabelExemptionsProto(accessibilityLabelExemptiontextProto: String):
+  AccessibilityLabelExemptions {
+    val protoBinaryFile = File("$accessibilityLabelExemptiontextProto.pb")
+    val builder = AccessibilityLabelExemptions.getDefaultInstance().newBuilderForType()
+
+    // This cast is type-safe since proto guarantees type consistency from mergeFrom(),
+    // and this method is bounded by the generic type T.
+    @Suppress("UNCHECKED_CAST")
+    val protoObj: AccessibilityLabelExemptions =
+      FileInputStream(protoBinaryFile).use {
+        builder.mergeFrom(it)
+      }.build() as AccessibilityLabelExemptions
+    return protoObj
+  }
```

Comment:
```suggestion
private fun loadAccessibilityLabelExemptionsProto(
  accessibilityLabelExemptiontextProto: String
): AccessibilityLabelExemptions {
  val protoBinaryFile = File("$accessibilityLabelExemptiontextProto.pb")
  val builder = AccessibilityLabelExemptions.getDefaultInstance().newBuilderForType()
   // This cast is type-safe since proto guarantees type consistency from mergeFrom(),
  // and this method is bounded by the generic type T.
  @Suppress("UNCHECKED_CAST")
  val protoObj: AccessibilityLabelExemptions =
    FileInputStream(protoBinaryFile).use {
      builder.mergeFrom(it)
    }.build() as AccessibilityLabelExemptions
  return protoObj
}
```

Prefer breaking at the arguments to avoid needing to indent the entire function body.

Reply:
Done.